exercises/concept/lucians-luscious-lasagna/.docs/hints.md

```diff
@@ -0,0 +1,31 @@
+# Hints
+
+## General
+
+-   An integer literal can be defined as one or more consecutive digits.
+
+## 1. Define the expected oven time in minutes
+
+-   You need to define a [function][functions] without any parameters.
+
+## 2. Calculate the remaining oven time in minutes
+
+-   You need to define a [function][functions] with a single parameter.
+-   You can use and refer to the previously defined item by its name.
+-   The last expression in a function is [automatically returned][return-values] from the function; you don't have to explicitly indicate which value to return.
+-   You can use the [mathematical operator for subtraction][operators] to subtract values.
+
+## 3. Calculate the preparation time in minutes
+
+-   You need to define a [function][functions] with a single parameter.
+-   You can use the [mathematical operator for multiplicaton][operators] to multiply values.
+
+## 4. Calculate the elapsed time in minutes
+
+-   You need to define a [function][functions] with two parameters.
+-   You can [call][functions] one of the other functions you've defined previously.
```

Comment:
I've removed the `[calling]` link reference and replaced it with the functions link as suggested.